examples/wind/package.json

```diff
@@ -20,10 +20,11 @@
   },
   "devDependencies": {
     "babel-core": "^6.21.0",
-    "buble-loader": "^0.4.0",
+    "babel-loader": "^6.2.10",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-2": "^6.18.0",
+    "buble-loader": "^0.4.0",
```

Comment:
I believe this can be remove altogether.

Reply:
no, when we use"npm run start-local", still need the bible-loader.